src/components/PokemonCard.js

```diff
@@ -0,0 +1,93 @@
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+import React from 'react';
+import usePokemon from './pokemon.js';
+
+export default function PokemonCard() {
+  const pokemon = usePokemon();
+  return (
+    <>
+      <div className="pokeContainer">
+        {pokemon.map((pkmn) => (
+          <div className="pokeCard" key={pkmn.id} >
+
+            <div className="image">
+              {pkmn.species_id.length === 3}
+              <img src={`http://assets.pokemon.com/assets/cms2/img/pokedex/full/${pkmn.species_id}.png`} />
+              {pkmn.species_id.length === 2}
+              <img src={`http://assets.pokemon.com/assets/cms2/img/pokedex/full/0${pkmn.species_id}.png`} />
+              {pkmn.species_id.length === 1}
+              <img src={`http://assets.pokemon.com/assets/cms2/img/pokedex/full/00${pkmn.species_id}.png`} />
+            </div>
```

Comment:
This is a great candidate for a `<PokemonImage />` component.

Reply:
Refactored and moved into PokemonImage file!